Address suggestions from WG for Helm docs (#8502)
istio/istio.io: content/en/docs/setup/install/helm/index.md

    $ kubectl get pods -n istio-system
    {{< /text >}}

## Updating your Istio configuration

You can provide override settings specific to any Istio Helm chart used above
and follow the Helm upgrade workflow to customize your Istio mesh installation.
The available configurable options can be found by inspecting the top level
`values.yaml` file associated with the Helm charts located at `manifests/charts`
inside the Istio release package specific to your version.

{{< warning >}}
Note that the Istio Helm chart values are under active development and
considered experimental. Upgrading to newer versions of Istio can involve
migrating your override values to follow the new API.
{{< /warning >}}

For customizations that are supported via both
[`ProxyConfig`](/docs/reference/config/istio.mesh.v1alpha1/#ProxyConfig) and Helm

> values, it is recommended to use `ProxyConfig` options as it provides schema
> validation over unstructured Helm values.

Reviewer: ```suggestion
values, using `ProxyConfig` is recommended because it provides schema
validation while unstructured Helm values do not.
```

[Operator uninstall guide](/docs/setup/install/operator/#uninstall)
depending upon your installation method.

> ### (Recommended) Canary Upgrade

Reviewer: ```suggestion
### Canary upgrade (recommended) 
```
